Karl/claire.h

```diff
@@ -277,7 +302,8 @@ class GenericClaire final {
     } catch (const current::Exception&) {
       last_keepalive_attempt_result_.status = KeepaliveAttemptStatus::CouldNotConnect;
     }
-    // TODO(dk+mz): Should it really throw in keepalive thread? It will crash the binary.
+    // TODO(dk+mz): Should it really throw in keepalive thread? It will
+    // crash the binary.
```

Comment:
Suppose this comment should be replaced with the one stating that there is a `catch` clause.